app/Lib/Rscripts/analyse.R

```diff
@@ -2,11 +2,31 @@
 library(psy)
 
 Analyse <- function(key, input.answers, number.answeroptions) {
+  # Calculates standard psychometric properties of an exam, notably the percentage correct and the item rest correlation (IRC) per item and 
+  # per answer option and the cronbach's alpha for the whole exam.
+  #
+  # Arguments:
```

Comment:
Check the Google's R Style Guide: https://google.github.io/styleguide/Rguide.xml#functiondefinition